### DISCUSSION TITLE
Offline  Status

### DISCUSSION THREAD

**Discussion by Yogesh-Dubey-Ayesavi** [xmppjs/xmpp.js#939] [Q&A] [answered]
How to appear as offline using this library ?
 I have processone Ejjaberd server setted up

**Comment by sonnyp** [comment, marked as answer]
```js
xmpp.on("online", () => {
  xmpp.send(<presence type="unvailable");
});

```

**Reply by Yogesh-Dubey-Ayesavi** [reply]
It works fine with most of the xmpp servers in the ejabberd i think there might be some technical issue server side otherwise thanks @sonnyp
